index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Nice job using `lang="en"` to the `Html` start tag, this is good for internationalization. You should check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics) on why it is important to use if you don’t know already. 👍 🌟